packages/smooth_app/lib/pages/product/nutrition_container.dart

```diff
@@ -187,8 +187,17 @@ class NutritionContainer {
   }
 
   /// Stores the nutrient [Unit].
-  void _setUnit(final String nutrientId, final Unit unit) =>
-      _units[_fixNutrientId(nutrientId)] = unit;
+  void _setUnit(
+    final String nutrientId,
+    final Unit unit, {
+    required final bool init,
```

Comment:
Could you document this field please?

Reply:
I don't believe in comments here:
* it's a private method
* the parameter has a decent name (`init`? "it must have something to do with the init phase")
* the method is used only twice, first time at init time with `init: true`, and later with `init: false`